src/models/Request.js

```diff
@@ -46,12 +46,17 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.ENUM,
       values: ['BUSINESS', 'VACATION', 'EXPEDITION'],
       defaultValue: 'BUSINESS'
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['OPEN', 'REJECTED', 'APPROVED'],
+      defaultValue: 'OPEN'
     }
   });
 
   Request.associate = models => {
     Request.belongsTo(models.User, {
-      foreignKey: 'userId',
+      foreignKey: 'UserId',
```

Comment:
why did you change all instances of `userId` to `UserId`

Reply:
I changed it because Sequelize was throwing an error that 'UserId' does not exist when I try to run the app. So when I changed 'userId' to UserId', the error stopped.